wgsl/Makefile

```diff
@@ -1,7 +1,13 @@
 all: index.html grammar/grammar.js
 
-index.html: index.bs
-	bikeshed --die-on=everything spec index.bs
+
+GHC=https://github.com/gpuweb/gpuweb/blob
+index.html: index.pre.html Makefile
+	# Append another line to the "This version:" metadata at the top.
+	sed -e "s,gpuweb/wgsl/</a>,gpuweb/wgsl/</a><br><a href=\"$(GHC)/$(shell git rev-parse HEAD)/wgsl/index.bs\">$(GHC)/$(shell git rev-parse HEAD)/wgsl/index.bs</a>," <index.pre.html >index.html
```

Comment:
only WGSL link? If we are doing this, let's do it for both specs

Reply:
Added to the WebGPU spec as well.   I'm not 100% sure this worked for the preview pages.